src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -449,6 +449,41 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 }
                 break;
 
+            case NI_Vector64_CreateScalarUnsafe:
+            case NI_Vector128_CreateScalarUnsafe:
+            {
+                if (intrin.op1->isContainedFltOrDblImmed())
+                {
```

Comment:
Nit: you don't need braces for the `case:` and we usually don't use them

Reply:
Actually we have mixed conventions. At some place we have them and others we don't. I noticed at [here](https://github.com/dotnet/runtime/blob/master/src/coreclr/src/jit/hwintrinsicarm64.cpp#L336) and [here](https://github.com/dotnet/runtime/blob/master/src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp#L391) so added them.

Reply:
In the second case the braces are needed - you need them if you allocate a local variable

Reply:
Yep, I realized it after merging. I will fix it in subsequent PT.